Mapper/BaseGroupedMapper.php

```diff
@@ -70,11 +70,17 @@ public function with($name, array $options = array())
             'collapsed' => false,
             'class' => false,
             'description' => false,
+            'label' => $this->admin->getLabelTranslatorStrategy()->getLabel($name, $this->getName(), 'group'),
             'translation_domain' => null,
```

Comment:
What about setting the `translation_domain` to `false`? This would follow symfony standard.